src/map/skill.cpp

```diff
@@ -19191,11 +19191,7 @@ bool skill_check_condition_castbegin( map_session_data& sd, uint16 skill_id, uin
 			}
 			break;
 		case ST_RECOVER_WEIGHT_RATE:
-#ifdef RENEWAL
-			if(pc_is70overweight(&sd)) {
-#else
-			if(pc_is50overweight(&sd)) {
-#endif
+			if( pc_getpercentweight(sd) >= battle_config.natural_heal_weight_rate ) {
 				clif_skill_fail( sd, skill_id );
```

Comment:
Did we confirm that this is the correct error message for TF_PICKSTONE and AC_MAKINGARROW in this case?

Reply:
Yeah I've checked and once you have the SC_WEIGHT50 status all skills that get you items, show the default skill fail message

Reply:
Yeah I've checked and once you have the SC_WEIGHT50 status all skills that get you items, show the default skill fail message